### PR TITLE
My take on Exercise 9.3.5

### PR DESCRIPTION
**Pull request by kcha193**
Hope your new year is going great. 

Here is what I think the answer for Exercise 9.3.5 should be. Please let me know what you think.

### CODE REVIEW

File: 09-uploads-downloads.Rmd [hunk 1]
```diff
@@ -4,10 +4,10 @@
 knitr::opts_chunk$set(eval = FALSE)
 ```
 
+## 9.4 Exercises {.unnumbered}
```
Comment (howardbaik) [inline 09-uploads-downloads.Rmd]: Is {.unnumbered} the same as {-}?

File: 09-uploads-downloads.Rmd [hunk 2]
```diff
@@ -150,8 +148,103 @@ server <- function(input, output) {
 shinyApp(ui, server)
 ```
 
+5.  From 9.3 Case study, the main changes is from the Clean step of server function, where one large reactive is broken down into three multiple smaller ones.
```
Comment (howardbaik) [inline 09-uploads-downloads.Rmd]:
```suggestion
5.  From the 9.3 Case study, the main change happens in the cleaning step inside the server function, where one large reactive is broken down into three smaller ones.
```

**Review by howardbaik — CHANGES_REQUESTED** [review]
Just a few minor comments, but overall, I agree with your approach to this answer. Thanks!

File: 09-uploads-downloads.Rmd (hunk 1 above)
Comment (kcha193) [reply 09-uploads-downloads.Rmd]: I dont remember I did this, it is likely got edited automatically when I switched to "visual markdown editor" mode in rstudio. Do you want me to change it back?

Comment (kcha193) [reply 09-uploads-downloads.Rmd]: Yea, I have changed it back.

**Comment by kcha193** [comment]
Many thanks, Howard, for your feedback and all your work on this. I will have a go on the other exercises when I have a bit of time. Have a good one!! :)

**Comment by howardbaik** [comment]
Thank you for contributing, Kevin. Please do have a go at the other exercises and submit a PR :)

**Pull request merged**